leakcanary-android-core/src/main/java/leakcanary/internal/tv/TvToast.kt

```diff
@@ -25,9 +25,9 @@ internal object TvToast {
     text: CharSequence
   ): Toast {
     val toast: Toast = Toast.makeText(activity, text, Toast.LENGTH_LONG)
-    val textView = toast.view.findViewById<TextView>(android.R.id.message)
-    textView.setCompoundDrawablesWithIntrinsicBounds(R.drawable.leak_canary_icon, 0, 0, 0)
-    textView.compoundDrawablePadding =
+    val textView = toast.view?.findViewById<TextView>(android.R.id.message)
```

Comment:
Starting with API 30 Toast.getView() returns null when the toast is constructed via `Toast.makeText()`. So we should likely switch to a full blown custom toast like in default LeakCanary?

@Armaxis thoughts on this?

Reply:
Wow, what an amazing API change, thanks, Google! /s

Yea, we should go with custom Toast then